### ISSUE TITLE
Attribute Error if sys.stdout/sys.stderr is None

### ISSUE THREAD

**Issue by Killah-Chicken** [bottlepy/bottle#1244]
If `sys.stdout is None or sys.stderr is None`, importing bottle causes an `AttributeError`.
It is caused by this piece of code **(bottle.py line 118)**, because _attribute write of NoneType could not be found_:

```python
# Workaround for the "print is a keyword/function" Python 2/3 dilemma
# and a fallback for mod_wsgi (resticts stdout/err attribute access)
try:
    _stdout, _stderr = sys.stdout.write, sys.stderr.write
except IOError:
    _stdout = lambda x: sys.stdout.write(x)
    _stderr = lambda x: sys.stderr.write(x)
```

I was wondering that `sys.stdout` could be set to `None`, but found:  [stackoverflow: stdout = None](https://stackoverflow.com/questions/49757674/why-does-sys-stdout-none-work)

I could fix it with setting and "empty" lambda to `_stdout` and `_stderr`
```python
try:
    _stdout, _stderr = sys.stdout.write, sys.stderr.write
except IOError:
    _stdout = lambda x: sys.stdout.write(x)
    _stderr = lambda x: sys.stderr.write(x)
except AttributeError:
    _stdout = _stderr = lambda x: None
```

**Comment by oz123** [comment]
I think one should consider using `from __future__ import print_function`.
At this point bottle only supports Python2.7 so it's safe. 

https://docs.python.org/2/library/__future__.html

**Comment by defnull** [comment]
Fixed in 5e2efb6a
